auth-web/src/components/auth/common/PADInfoForm.vue

```diff
@@ -126,169 +123,155 @@
 </template>
 
 <script lang="ts">
-import { Component, Emit, Mixins, Prop, Vue } from 'vue-property-decorator'
-import { mapMutations, mapState } from 'vuex'
-import { Account } from '@/util/constants'
+import { ComputedRef, computed, defineComponent, nextTick, onMounted, reactive, ref, toRefs, watch } from '@vue/composition-api'
 import CommonUtils from '@/util/common-util'
 import { PADInfo } from '@/models/Organization'
 import TermsOfUseDialog from '@/components/auth/common/TermsOfUseDialog.vue'
 import { mask } from 'vue-the-mask'
+import { useStore } from 'vuex-composition-helpers'
 
-@Component({
-  directives: {
-    mask
-  },
-  components: {
-    TermsOfUseDialog
-  },
-  computed: {
-    ...mapState('org', [
-      'currentOrgPADInfo',
-      'currentOrganizationType'
-    ])
-  },
-  methods: {
-    ...mapMutations('org', [
-      'setCurrentOrganizationPADInfo'
-    ])
-  }
-})
-export default class PADInfoForm extends Vue {
-  @Prop({ default: () => ({} as PADInfo) }) padInformation: any
-  @Prop({ default: false }) isChangeView: boolean
-  @Prop({ default: true }) isAcknowledgeNeeded: boolean
-  @Prop({ default: true }) isTOSNeeded: boolean
-  @Prop({ default: false }) isInitialTOSAccepted: boolean
-  @Prop({ default: false }) isInitialAcknowledged: boolean
-  @Prop({ default: false }) clearOnEdit: boolean
-  private readonly currentOrgPADInfo!: PADInfo
-  private readonly currentOrganizationType!: string
-  private readonly setCurrentOrganizationPADInfo!: (padInfo: PADInfo) => void
-  private transitNumber: string = ''
-  private institutionNumber: string = ''
-  private accountNumber: string = ''
-  private isTOSAccepted: boolean = false
-  private isAcknowledged: boolean = false
-  private isTouched: boolean = false
-  private isStartedEditing: boolean = false
-  private bankInfoDialog: boolean = false
+// FUTURE: remove this in vue 3
+interface PADInfoFormState {
```

Comment:
Should we put this in a separate interface folder?

Reply:
I was thinking we would keep them inside since it'll be deleted asap rather than clutter the interfaces folder. It should be inferring the types without the interface (every other project I've tried you don't need this). It's a symptom to do with your comment below